src/features/user/Settings/ImpersonateUser/ImpersonationActivated.tsx

```diff
@@ -27,8 +27,8 @@ const ImpersonationActivated = () => {
 
   return isImpersonationModeOn ? (
     <div className={styles.impersonationAlertContainer}>
-      <p className={styles.impersonatingText}>{t('me:targetUser')}</p>
-      <p className={styles.impersonatingEmail}>{`<${user.email}>`}</p>
+      <div className={styles.impersonatingText}>{t('me:targetUser')}</div>
+      <div>{`<${user.email}>`}</div>
```

Comment:
@sunilsabatp You don't need to keep this text in 2 separate divs. Just use regular interpolation and combine the strings. Keeping 2 separate divs for the text results in the text in both divs responding separately to different screen sizes.